jlm/llvm/opt/alias-analyses/Steensgaard.cpp

```diff
@@ -24,7 +24,8 @@ namespace jlm::llvm::aa
 static bool
 HasOrContainsPointerType(const rvsdg::output & output)
 {
-  return IsOrContains<PointerType>(output.type());
+  return IsOrContains<PointerType>(output.type())
+      || dynamic_cast<const llvm::FunctionType *>(&output.type());
```

Comment:
`is<FunctionType>()`

Also, the name of the function does not really reflect its body any longer

Reply:
see above